src/inc/CMakeLists.txt

```diff
@@ -70,6 +70,7 @@ add_library(corguids ${CORGUIDS_SOURCES})
 
 install (FILES cor.h
                corhdr.h
+               corhost.h
```

Comment:
I published it because it contain CorHost2 hosting interface, but I see now that it is the private implementation.

mikem

From: Jan Kotas [mailto:notifications@github.com]
Sent: Monday, June 1, 2015 3:04 PM
To: dotnet/coreclr
Cc: Mike McLaughlin
Subject: Re: [coreclr] General SOS support cleanup (#1080)


In src/inc/CMakeLists.txt<https://github.com/dotnet/coreclr/pull/1080#discussion_r31475513>:

> @@ -70,6 +70,7 @@ add_library(corguids ${CORGUIDS_SOURCES})

>

>  install (FILES cor.h

>                 corhdr.h

> +               corhost.h

corhost.h is internal implementation detail. We should not be dropping it for packaging. What is it needed for?

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/coreclr/pull/1080/files#r31475513>.